seedlet/inc/wpcom.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * WordPress.com-specific functions and definitions.
+ *
+ * This file is centrally included from `wp-content/mu-plugins/wpcom-theme-compat.php`.
+ *
+ * @package Seedlet
+ */
+
+/**
+ * Add setting for hiding page title on the homepage.
+ */
+function seedlet_wpcom_customize_update( $wp_customize ) {
+	$wp_customize->add_setting( 'hide_front_page_title', array(
+		'default'              => false,
+		'type'                 => 'theme_mod',
+		'transport'            => 'postMessage',
+		'sanitize_callback'    => 'seedlet_sanitize_checkbox',
+	) );
+
+	$wp_customize->add_control( 'hide_front_page_title', array(
+		'label'		  => esc_html__( 'Hide Homepage Title', 'seedlet' ),
+		'description' => esc_html__( 'Check to hide the page title, if your homepage is set to display a static page.', 'seedlet' ),
+		'section'	  => 'static_front_page',
+		'priority'	  => 10,
+		'type'		  => 'checkbox',
+		'settings'	  => 'hide_front_page_title',
+	) );
+}
+add_action( 'customize_register', 'seedlet_wpcom_customize_update' );
+
+/**
+* Sanitize the checkbox.
+*
+* @param boolean $input.
+*
+* @return boolean true if is 1 or '1', false if anything else
+*/
+function seedlet_sanitize_checkbox( $input ) {
+	if ( 1 == $input ) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+/**
+ * Bind JS handlers to instantly live-preview changes.
+ */
+function seedlet_wpcom_customize_preview_js() {
+	wp_enqueue_script( 'seedlet_wpcom_customize_preview', get_theme_file_uri( '/inc/customize-preview-wpcom.js' ), array( 'customize-preview' ), '1.0', true );
+}
+add_action( 'customize_preview_init', 'seedlet_wpcom_customize_preview_js' );
+
+/**
+ * Enqueue our WP.com styles for front-end.
+ * Loads after style.css so we can add overrides.
+ */
+function seedlet_wpcom_scripts() {
+	wp_enqueue_style( 'seedlet-wpcom-style', get_template_directory_uri() . '/inc/style-wpcom.css', array( 'seedlet-style' ), '20200629' );
+}
+add_action( 'wp_enqueue_scripts', 'seedlet_wpcom_scripts' );
+
+/**
+ * Adds custom classes to the array of body classes.
+ *
+ * @param array $classes Classes for the body element.
+ * @return array
+ */
+function seedlet_wpcom_body_classes( $classes ) {
+
+	$hide = get_theme_mod( 'hide_front_page_title', false );
+
+	if ( true === $hide ) {
+		$classes[] = 'hide-homepage-title';
+	}
+
+	$credit_option = get_option( 'footercredit' );
```

Comment:
This is if the user is on a business plan and has selected "Hide", we add a class to hide the comma?

<img width="292" alt="Screen Shot 2020-06-30 at 10 29 35 AM" src="https://user-images.githubusercontent.com/5375500/86138569-a39fd380-babc-11ea-840d-85e8c063f5f6.png">

Reply:
Yep, that's right.